website/src/blogs/migrate-cstorpoolinstances-cspis-to-different-node-by-moving-the-disks.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Migrate CSPIs to a different node by moving the disks
+author: Sai Chaithanya
+author_info: A developer whos is always eager to learn, loves algorithms, maths, Kubernetes, and programming, Passionate towards Data Science. Enjoys playing kabaddi and traveling.
+date: 04-11-2020
+tags: OpenEBS
+excerpt: Step by step guide to migrate CStorPoolInstances from one node to different nodes by moving the set of underlying disks
+---
+
+
+This blog describes steps to migrate CStorPoolInstances from one node to different nodes by **moving the set of underlying disks to a new node that participates in the pool provisioning**. There were a couple of use cases where this feature can be helpful:
+
+1. Scaling down and scaling up nodes in the cluster(in a cloud environment) by retaining external volumes(for cost savings).
+2. Replacing failed storage nodes with new nodes by attaching the same old disks to the new node.
+
+**Steps to migrate the CSPI to different node:**
+
+1. Detach the disks belonging to the CSPI that you wish to migrate from the node and attach it to the new node. If you are using a cloud platform, check on their documentation, or ask the administrator about the steps to do this.
+2. Change the node selector in the CSPC YAML (next section describes how to do this).
+
+![](https://lh4.googleusercontent.com/XTwKu6lE3lyoZ3cHRO9HNJGUaTOoGfE-OWGuscrmukbxEKJNPSaEqxUPbbNnnc3dcD-Aybc2_AF0y2Scf0QBxSDG_f9QZWRu67sXZjoMKO6nymhgelEWfDzPjfGKi4D9UwLBaN0D)
```

Comment:
What are we planning about this type of `IMG block`?

Reply:
This we will import as it is till we get the proper images.